docs/scripts/basic.md

```diff
@@ -0,0 +1,70 @@
+## Welcome
+
+**Facilitator**
+Welcome everyone. Thank you so much for taking the time join us together so that we can all have a more satisfying 
+democracy.
+
+## Introductions
+
+**Facilitator**
+Let's begin by sharing our names and pronouns. Also, what brought you here. I'll start first.
+
+My name is Ben. The pronouns I use are "he", "him" and "his" as well as "they", "them" and "theirs". What brings me here is the desire to see a world that works for everyone where we can all take each other seriously.
+
+*Everyone shares their names, pronouns and purpose.*
+
+## Process & Purpose
+
+Our main objective for the today is to come up with meaningful questions to ask the people whom we share our planet with. Some of these people are those who run for office and we are negotiating expectations with. Some are currently in office and we must help them uphold the will of the people. We shall ask questions to activists that increase our mutual understanding of each other. We may also need to help each other understand how different people come to the conclusions that they make and how to be free to make decisions that we can all be proud of.
+
+We shall first plan together based on the complete interest of the group what we want to work on today.
+We shall then make a plan for how we want to work together today.
+Using our plan we shall practice investigating and researching different ways the issue can be approached.
+We shall make up questions that we could ask to specific people that might help us make places that we all want to live together in. 
```

Comment:
We shall explore questions that we could ask to specific people that might help us create an environment that we all want to live in and explore together.